.github/workflows/tools-api-docs-dev.yml

```
@@ -26,18 +26,14 @@ jobs:
       - name: Build HTML docs
         run: make --directory ./docs/api html
 
-      - name: Push docs to api-doc branch
+      - name: Sync dev docs
         if: github.repository == 'nf-core/tools'
-        run: |
-          git checkout --orphan api-doc
-          git rm -r --cache .
-          rm .gitignore
-          git config user.email "core@nf-co.re"
-          git config user.name "nf-core-bot"
-          git add docs
-          git commit --message "nf-core/tools docs build
-
-          $GITHUB_REF - $GITHUB_SHA
-          "
-          git remote add nf-core https://github.com/nf-core/tools.git
-          git push --force --set-upstream nf-core api-doc
+        uses: SamKirkland/FTP-Deploy-Action@4.0.0
+        with:
+          server: ${{ secrets.ftp_server }}
+          username: ${{ secrets.ftp_username}}
+          password: ${{ secrets.ftp_password }}
+          local-dir: docs
```

https://github.com/SamKirkland/FTP-Deploy-Action#settings

> _Folder to upload from, must end with trailing slash `/`_

```suggestion
          local-dir: './docs/'
```